modules/extension/xsd/xsd-core/src/main/java/org/geotools/xml/PullParser.java

```diff
@@ -14,6 +14,8 @@
 import org.geotools.xml.impl.ElementHandler;
```

Comment:
We normally add/fix headers as we notice them missing or outdated, so yes, it would be best if it was  part of this commit :-)